clans/src/main/java/me/mykindos/betterpvp/clans/clans/ClanManager.java

```diff
@@ -152,6 +154,52 @@ public boolean hasAccess(Player player, Location location) {
         return relation == ClanRelation.SELF || relation == ClanRelation.ALLY_TRUST;
     }
 
+    public Location closestWilderness(Player player) {
+        int maxChunksRadiusToScan = 3;
+        List<Chunk> chunks = new ArrayList<>();
+
+        Chunk playerChunk = player.getChunk();
+        World world = player.getWorld();
+
+        boolean end = false;
+
+        for (int i = 1; i < maxChunksRadiusToScan; i++) {
+            int[] offset = {-i, 0, i};
+            for (int x : offset) {
+                for (int z: offset) {
```

Comment:
For each `i` until `maxChunksRadiusToScan` you're always scanning the player's own chunk because one of the elements in the `offset` array is 0, meaning at one point both x and z will both be zero and pointlessly check the player's chunk.

Just replace these three for loops with two that iterate from 0 to maxChunksRadiusToScan and have one of them be x (or i) and z (or j)

Something like this:
`
for (int i = 0; i < maxChunksRadiusToScan; i++) {
    for (int j = 0; j < maxChunksRadiusToScan; j++) {
        // get the chunk (i, j)
    }
}
`

Reply:
Wouldn't this still not give chunks in the negative directions relative to the player? I would think it might be better to just check if x == 0 and z == 0 and then skip.

Reply:
You're right, this fixes that:

```
for (int i = -maxChunksRadiusToScan; i < maxChunksRadiusToScan; i++) {
    for (int j = -maxChunksRadiusToScan; j < maxChunksRadiusToScan; j++) {
        // get the chunk (i, j)
    }
}
```

Reply:
Implemented